react/index.js

```diff
@@ -286,7 +286,31 @@ class ProductSummary extends Component {
       'bb b--muted-4 ma2': showBorders
     })
 
-    const style = { visibility: showBuyButton ? 'visible' : 'hidden' }
+    const buyButtonStyle = { visibility: showBuyButton ? 'visible' : 'hidden' }
+
+    const renderBuyButton = () => (
```

Comment:
This is a render task, do not you think that should be better to extract this to a function?